_config.yml

```diff
@@ -122,6 +122,37 @@ redirects:
   /it/bitcoin-per-appassionati: /it/innovazione
   /nl/bitcoin-voor-liefhebbers: /nl/innovatie
   /pl/bitcoin-dla-entuzjastow: /pl/innowacje
+  ## Old releases pages; stop adding to this list after 0.10.0
+  /releases/2011/04/27/v0.3.21: /en/release/v0.3.21
+  /releases/2011/06/05/v0.3.22: /en/release/v0.3.22
+  /releases/2011/06/14/v0.3.23: /en/release/v0.3.23
+  /releases/2011/07/08/v0.3.24: /en/release/v0.3.24
+  /releases/2011/09/23/v0.4.0: /en/release/v0.4.0
+  /releases/2011/11/21/v0.5.0: /en/release/v0.5.0
+  /releases/2011/12/15/v0.5.1: /en/release/v0.5.1
+  /releases/2012/01/09/v0.5.2: /en/release/v0.5.2
+  /releases/2012/03/14/v0.5.3: /en/release/v0.5.3
+  /releases/2012/03/16/v0.5.3.1: /en/release/v0.5.3.1
+  /releases/2012/03/30/v0.6.0: /en/release/v0.6.0
+  /releases/2012/05/04/v0.6.1: /en/release/v0.6.1
+  /releases/2012/05/08/v0.6.2: /en/release/v0.6.2
+  /releases/2012/06/25/v0.6.3: /en/release/v0.6.3
+  /releases/2012/09/17/v0.7.0: /en/release/v0.7.0
+  /releases/2012/10/19/v0.7.1: /en/release/v0.7.1
+  /releases/2012/12/14/v0.7.2: /en/release/v0.7.2
+  /releases/2013/02/19/v0.8.0: /en/release/v0.8.0
+  /releases/2013/03/18/v0.8.1: /en/release/v0.8.1
+  /releases/2013/05/29/v0.8.2: /en/release/v0.8.2
+  /releases/2013/06/25/v0.8.3: /en/release/v0.8.3
```

Comment:
@harding According to the stats, you could stop at 0.8.2 (the old links were generated but never referenced at this point):
https://bitcoin.org/stats/ (you can sort by urls)